nsv13/code/modules/clothing/helmet_camera.dm

```diff
@@ -64,3 +59,26 @@
 		updating = TRUE
 		addtimer(CALLBACK(src, PROC_REF(do_camera_update), oldLoc), SILICON_CAMERA_BUFFER)
 #undef SILICON_CAMERA_BUFFER
+
+/obj/item/clothing/head/helmet/proc/install_camera(obj/item/I, mob/user, faction)
+	if(builtInCamera) //You already have a camera
+		return FALSE
+	if(!I || !user) //The camera isn't being placed by a person
+		builtInCamera = new /obj/machinery/camera(src)
+		builtInCamera.c_tag = "Helmet Cam #[rand(0,999)]"
+		if(faction == "Syndicate")
+			builtInCamera.network = list("syndicate")
+		else
+			builtInCamera.network = list("ss13")
+			builtInCamera.internal_light = FALSE
+		return TRUE
+	user.transferItemToLoc(I, src)
+	builtInCamera = new /obj/machinery/camera(src)
+	builtInCamera.c_tag = "Helmet Cam #[rand(0,999)]"
+	if(user.faction & "Syndicate" || faction == "Syndicate") //If a syndicate agent puts it on, it's a syndie camera now
```

Comment:
This won't work as expected, as first of all `&` binds very weakly so it should be contained within parentheses, and additionally, `&` when used with a list always gives back a list, which means the condition is always true (as an empty list evaluates as true within if). You will have to either `length()` check the result, or use `in` or a comparable operation instead.